plugins/woocommerce-blocks/tests/e2e/tests/checkout/checkout-block.shopper.block_theme.side_effects.spec.ts

```diff
@@ -474,3 +474,37 @@ test.describe( 'Checkout Form Errors', () => {
 		).toBeVisible();
 	} );
 } );
+
+test.describe( 'Billing Address Form', () => {
+	test.use( { storageState: guestFile } );
+	const shippingTestData = {
```

Comment:
As discussed over the huddle, we could make this an array of addresses to test multiple edge cases and make it more robust 👍🏼

Reply:
@wavvves I appreciate your suggestion to iterate through the fields to ensure that all of them are checked. However, I tried using a loop and encountered a limitation with the field type on the front-end. Specifically, the country and state fields require a `toHaveText` assertion check, which results in conditional assertions that are not recommended by our linters.

I also came to the realization that there is no need for the loop to iterate the fields because they will always be the same as the shipping address fields. Prior to the test, we define the shipping address, so we already know which address fields will be available for the billing address form. I'd love to hear your thoughts about it.